src/Offcanvas.tsx

```diff
@@ -272,12 +280,11 @@ const Offcanvas: BsPrefixRefForwardingComponent<'div', OffcanvasProps> =
 
       const renderDialog = (dialogProps) => (
         <div
-          role="dialog"
```

Comment:
Removed this because the role is passed through dialogProps by @restart/ui anyways.